README.md

```diff
@@ -39,6 +39,15 @@
     - run: **python main.py all** to run both the generation and simulation.
     - run: **python main.py [simulate|generate] --help** for more help about each option.
     
+    
+## Installing and running under Linux (tested on Debian 10):
+- sudo apt install swig liblzma-dev libbz2-dev
+    - swig - building parasymoblic matrix needs swig
```

Comment:
Using swig will require the users to also install swimport, which is a dev-only requirement. We cannot expect the client to install swig.

Reply:
Let’s release the compiled binaries to GitHub.
https://stackoverflow.com/questions/5207269/how-to-release-a-build-artifact-asset-on-github-with-a-script

Reply:
What's bad about making swimport and swig as regular dependencies instead of releasing binaries?